src/golang/cmd/server/server/execute_handler.go

```diff
@@ -25,15 +25,7 @@ func HandleError(
 	}
 	server.Log(ctx, handlerName, r, statusCode, err)
 
-	var externalMsg string
-	dbxErr, ok := err.(errors.DropboxError)
-	if ok {
-		// Only return the top-level error message to clients.
-		externalMsg = dbxErr.GetMessage()
-	} else {
-		externalMsg = err.Error()
-	}
-	response.SendErrorResponse(w, externalMsg, statusCode)
+	response.SendErrorResponse(w, errors.GetMessage(err), statusCode)
```

Comment:
@kenxu95 I found that `errors.GetMessage(err)` has the same functionality as the original code, with the added benefit that it gets errors at all levels instead of just the top-level, which I found to be much more useful both to ourselves and to the users.

Reply:
Actually since this is such a useful and scoped changed, can you make a new PR with just this? I can stamp it immediately (might want it for my error handling work)